dpsim/src/PFSolver.cpp

```diff
@@ -321,6 +321,19 @@ void PFSolver::determineNodeBaseVoltages() {
             "Choose base voltage {}V of {} to convert pu-solution of {}.",
             baseVoltage_, gen->name(), node->name());
         break;
+      } else if (std::shared_ptr<CPS::SP::Ph1::Load> load =
+                     std::dynamic_pointer_cast<CPS::SP::Ph1::Load>(comp)) {
+        baseVoltage_ = load->attributeTyped<CPS::Real>("V_nom")->get();
+        mSLog->info("Choose base voltage of {}V to convert pu-solution of {}.",
```

Comment:
Maybe is better with `SPDLOG_LOGGER_INFO(`

Reply:
Yes, the macro variant can be optimized better if the log level is less than info.